quay-enterprise/tectonic/index.md

```diff
@@ -0,0 +1,89 @@
+# Quay Enterprise Installation on Tectonic
+
+This guide walks through the deployment of [Quay Enterprise][quay-enterprise-tour] onto a Tectonic cluster.
+After completing the steps in this guide, a deployer will have a functioning instance of Quay Enterprise orchestrated as a Kubernetes service on a Tectonic cluster, and will be able to access the Quay Enterprise Setup tool with a browser to complete configuration of image repositories, builders, and users.
+
+[quay-enterprise-tour]: https://quay.io/tour/enterprise
+
+## Prerequisites
+
+A PostgreSQL database must be available for Quay Enterprise metadata storage.
+We currently recommend running this database server outside of the cluster.
+
+## Download Kubernetes Configuration Files
+
+Visit your [Tectonic Account][tectonic-account] and download the pre-formatted pull secret, under "Account Assets". There are several formats of the secret, be sure to download the Kubernetes-formatted file. This pull secret is used to download the Quay Enterprise containers.
```

Comment:
Visit your [Tectonic Account][tectonic-account] and download the pre-formatted pull secret, under "Account Assets". There are several formats of the secret, be sure to download the "dockercfg" format resulting in a `config.json` file. This pull secret is used to download the Quay Enterprise containers.